Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst

```diff
@@ -50,6 +76,49 @@ Example of a :php:`CustomSource` implementation:
     :language: php
     :caption: EXT:my_extension/Classes/Redirects/CustomSource.php
 
+Default event listeners
+=======================
+
+The listener :php:`\TYPO3\CMS\Redirects\EventListener\AddPageTypeZeroSource`
+creates a :php:`\TYPO3\CMS\Redirects\RedirectUpdate\PageTypeSource` for a page
+before the slug has been changed. The full URI is built to fill the `source_host`
+and `source_path`, which takes configured `RouteEnhancers` and `RouteDecorators`
+into account, e.g. the `PageType route decorator`.
+
+..  note::
+
+    If `source_host` and `source_path` lead to the same outcome for page type 0
+    using the full URI building like the
+    :php:`\TYPO3\CMS\Redirects\RedirectUpdate\PlainSlugReplacementSource`, the
+    :php:`PlainSlugReplacementSource` is replaced with the :php:`PageTypeSource`.
+
+It is not possible to configure for which page types sources should be added. If
+you need to do so, read :ref:`additional `PageTypeSource` auto-create redirect source type <feature-94499-1675615570>`
```

Comment:
This reference does not make sense here at is refers to this page. Instead, it should be linked to the example above.